source/unified-test-format/unified-test-format.rst

```diff
@@ -369,6 +369,13 @@ The structure of this object is as follows:
   "sharded" topology, test runners MUST accept any type of sharded cluster (i.e.
   "sharded" implies "sharded-replicaset", but not vice versa).
 
+- ``serverless``: Optional string. Whether or not the test should be run on
+  Serverless instances imitating sharded clusters. Valid values are "require",
+  "forbid", and "allow". If "require", the test MUST only be run on Serverless
+  instances. If "forbid", the test MUST only be run on actual sharded
+  deployments. If omitted or "allow", the test can be run on either Serverless
```

Comment:
> If "forbid",  the test MUST NOT be run on Serverless instances. If omitted or "allow", this option has no effect. 
 
This will future-proof the text in case something else comes along that masquerades as a mongos. Also, I think "no effect" is easier to understand for anyone implementing this (and also avoids enumerating the current alternatives).

Reply:
updated